ui/src/layout/Findings/utils.js

```diff
@@ -41,8 +29,8 @@ export const FindingsDetailsCommonFields = ({foundOn, invalidatedOn}) => (
         </TitleValueDisplayRow>
         {!!invalidatedOn &&
             <TitleValueDisplayRow>
-                <TitleValueDisplay title="Replaced by a newer scan on">{formatDate(invalidatedOn)}</TitleValueDisplay>
+                <TitleValueDisplay title="Invalidated on">{formatDate(invalidatedOn)}</TitleValueDisplay>
```

Comment:
was it wrong? not sure what "invalidated" means to the user

Reply:
"Replaced by a new scan" to me means there was a scan which caused the finding to be invalidated, this might not be true if we update the finding graph in the future dynamically from the vulnerability databases etc.

"Replaced" also implies something was created to replace it, which also might not be true, a finding might be marked invalidated because it doesn't exist any more.